### PR TITLE
Add redirect for modular contracting

### PR DESCRIPTION
**Pull request by lauraGgit**
- in part of the modular contracting sunset - noted in the draft post here: https://federalist-proxy.app.cloud.gov/preview/18f/18f.gsa.gov/modular-contracting/2019/04/09/why-we-love-modular-contracting/,

We will be archiving the site in favor of the repository

### CODE REVIEW

**Comment by eddietejeda** [comment]
@amirbey We need to change https://github.com/18F/dns, right?

**Comment by amirbey** [comment]
yes @eddietejeda that is correct .... a service has to be created ... i will raise a PR todo this ...

@lauraGgit - can this site be momentarily disabled (~15 mins)? .. i may have to momentarily remove modularcontracting.18f.gov from 18f/dns in order to create the service in cloud.gov

**Comment by lauraGgit** [comment]
@amirbey downtime is ok!

File: templates/_federalist-redirects.njk [hunk 1]
```diff
@@ -203,6 +203,14 @@ server {
...
 }
 
+# modularcontracting.18f.gov to https://github.com/18F/Modular-Contracting-And-Agile-Development
+server {
+  listen {{ PORT }};
+  set $target_domain github.com/18F/Modular-Contracting-And-Agile-Development;
+  server_name modularcontracting.18f.gov;
+  return 301 https://$target_domain;
+}
+
```
Comment (jmhooper) [inline templates/_federalist-redirects.njk]: Looks like we may also need to add a redirect for state-faq to go to GitHub

Comment (amirbey) [reply templates/_federalist-redirects.njk]: @jmhooper  - wouldn't state-faq redirect to modularcontracting.18f.gov which will redirect github?

Comment (amirbey) [reply templates/_federalist-redirects.njk]: fyi, @jmhooper agrees there is no need to add a redirect for state-faq to go to GitHub

**Comment by amirbey** [comment]
note: a custom-domain was used instead of a cdn-route which is typically used

**Pull request merged**